### PR TITLE
Isrumnon patch 1

### PR DESCRIPTION
**Pull request by isrumnon**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

File: skype/skype-ps/skype/Get-CsOnlineVoiceUser.md [hunk 1]
```diff
@@ -33,6 +33,20 @@ PS C:\> Get-CsOnlineVoiceUser -Identity Ken.Myer@contoso.com
 
 This example uses the User Principal Name (UPN) to retrieve the location and phone number information.
 
+**NOTE**
```
Comment (dariomws) [inline skype/skype-ps/skype/Get-CsOnlineVoiceUser.md:36]:
```suggestion
**NOTE:**
```

File: skype/skype-ps/skype/Get-CsOnlineVoiceUser.md [hunk 2]
```diff
@@ -33,6 +33,20 @@ PS C:\> Get-CsOnlineVoiceUser -Identity Ken.Myer@contoso.com
...
+
+Beginning TPM 2.6.2 onwards, the below updates are applicable for TeamsOnly customers
```
Comment (dariomws) [inline skype/skype-ps/skype/Get-CsOnlineVoiceUser.md:38]:
```suggestion
Beginning TPM 2.6.2 onwards, the below updates are applicable for TeamsOnly customers.
```

File: skype/skype-ps/skype/Get-CsOnlineVoiceUser.md [hunk 3]
```diff
@@ -33,6 +33,20 @@ PS C:\> Get-CsOnlineVoiceUser -Identity Ken.Myer@contoso.com
 
 This example uses the User Principal Name (UPN) to retrieve the location and phone number information.
 
+**NOTE**
+
+Beginning TPM 2.6.2 onwards, the below updates are applicable for TeamsOnly customers 
+
+*Deprecated Attributes*
+
+- Below attributes have been deprecated under "-ExpandLocation" parameter
```
Comment (dariomws) [inline skype/skype-ps/skype/Get-CsOnlineVoiceUser.md:42]:
```suggestion
- Below attributes have been deprecated under "-ExpandLocation" parameter:
```

**Review by dariomws — CHANGES_REQUESTED** [review]
Please move this note from "Examples" section to "Description". Thanks.

**Comment by get-itips** [comment]
Hello @isrumnon Isaiah,
As @dariomws suggests, please move this to the Description section

image

**Comment by isrumnon** [comment]
Hi Andres and Dario,

I've made the suggested changes

Thanks

**Review by dariomws — APPROVED** [review]
Thank you @isrumnon.

I think it's a good idea to replace TPM with "Microsoft Teams PowerShell version" or "Microsoft Teams PowerShell Module version" as @get-itips suggested.
With just TPM we can expect that someone opens an issue to ask about what the acronym means.

**Comment by isrumnon** [comment]
Done

**Review by tseward — APPROVED** [review]

File: skype/skype-ps/skype/Get-CsOnlineVoiceUser.md [hunk 4]
```diff
@@ -23,6 +23,20 @@ Get-CsOnlineVoiceUser [-CivicAddressId <XdsCivicAddressId>] [-DomainController <
...
+
+*Deprecated Attributes*
+
+- Below attributes have been deprecated under "-ExpandLocation" parameter:
+  - Force
+  - NumberOfResultsToSkip
+  - CorrelationId
+  - Verb
+  - ResultSize
+
```
Comment (get-itips) [inline skype/skype-ps/skype/Get-CsOnlineVoiceUser.md:38]:
```suggestion
```

File: skype/skype-ps/skype/Get-CsOnlineVoiceUser.md [hunk 5]
```diff
@@ -23,6 +23,20 @@ Get-CsOnlineVoiceUser [-CivicAddressId <XdsCivicAddressId>] [-DomainController <
...
+- LiscenceState
```
Comment (get-itips) [inline skype/skype-ps/skype/Get-CsOnlineVoiceUser.md:39]:
```suggestion
- LicenceState 
```

**Comment by yogkumgit** [comment]
@get-itips

File: skype/skype-ps/skype/Get-CsOnlineVoiceUser.md [hunk 6]
```diff
@@ -23,6 +23,19 @@ Get-CsOnlineVoiceUser [-CivicAddressId <XdsCivicAddressId>] [-DomainController <
...
+
+*Deprecated Attributes*
+
+- Below attributes have been deprecated under "-ExpandLocation" parameter:
+  - Force
+  - NumberOfResultsToSkip
+  - CorrelationId
+  - Verb
+  - ResultSize
+- LicenceState
```
Comment (get-itips) [inline skype/skype-ps/skype/Get-CsOnlineVoiceUser.md:38]: can you please fix the indentation?

Comment (isrumnon) [reply skype/skype-ps/skype/Get-CsOnlineVoiceUser.md:38]: This indentation is intentional, line 35,36 and 37 are sub points while 38 is a point by itself

Comment (get-itips) [reply skype/skype-ps/skype/Get-CsOnlineVoiceUser.md:38]: ok

**Review by get-itips — APPROVED** [review]

**Pull request merged**